analyses/comparative-RNASeq-analysis/01-correlation-matrix.py

```diff
@@ -0,0 +1,107 @@
+
+"""
+Create correlation matrices for polyA samples and ribodeplete (stranded) samples using gene expression data.
+The correlations will be calculated using pairwise Spearman correlation of the RNA-Seq gene expression profiles.
+The gene expression profile data will be filtered using a developed method described in Vaske et al.
+
+https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/229
+"""
+
+import os
+import sys
+import argparse
+import math
+import numpy as np
+import pandas as pd
+import scipy.stats
+import sklearn.metrics.pairwise as sklp
+import utils
+
+def prepare_expression(input_matrix,
+    proportion_unexpressed=0.8,
+    variance_filter_level=0.2,
```

Comment:
Should these be added as optional arguments that can be passed from the command line?

Reply:
Done.